tests/tests.cpp

```diff
@@ -3,6 +3,8 @@
 #include <catch2/catch.hpp>
 #include "delaunay.h"
 
+using namespace dt;
```

Comment:
Not a fan of using `using` in files, especially if the namespace is 2 characters.

Reply:
Oops, actually I don't know why I didn't just move the tests into the namespace too -> adjusted.